x/dispensation/keeper/executors.go

```diff
@@ -33,28 +34,28 @@ func (k Keeper) CreateDrops(ctx sdk.Context, output []bank.Output, name string)
 // It checks if any pending records are present , if there are it completes the top 10
 func (k Keeper) DistributeDrops(ctx sdk.Context, height int64) error {
 	pendingRecords := k.GetPendingRecordsLimited(ctx, 10)
-	for _, record := range pendingRecords {
-		err := k.GetSupplyKeeper().SendCoinsFromModuleToAccount(ctx, types.ModuleName, record.RecipientAddress, record.Coins)
+	for _, record := range pendingRecords.DistributionRecords {
+		err := k.GetBankKeeper().SendCoinsFromModuleToAccount(ctx, types.ModuleName, sdk.AccAddress(record.RecipientAddress), record.Coins)
 		if err != nil {
-			return errors.Wrapf(types.ErrFailedOutputs, "for address  : %s", record.RecipientAddress.String())
+			return errors.Wrapf(types.ErrFailedOutputs, "for address  : %s", record.RecipientAddress)
 		}
-		record.ClaimStatus = types.Completed
-		record.DistributionCompletedHeight = height
-		err = k.SetDistributionRecord(ctx, record)
+		record.ClaimStatus = sdk.NewUint(2)
```

Comment:
why use integers instead of the enum value generated by proto. If someone changes the enum values this could lead to a hard to find bug.